dataactbroker/routeUtils.py

```diff
@@ -28,6 +28,8 @@ def run_instance_function(accountManager, accountFunction, getSystemEmail = Fals
                 else:
                     # Currently no functions with user but not credentials flag
                     raise ValueError("Invalid combination of flags to run_instance_function")
+            elif(addTrueFlag):
```

Comment:
Simplified run_instance_function to just pass through an argument list, it's much cleaner now.